### PR TITLE
Alarm player branch

### PR DESCRIPTION
**Pull request by TY-tianyu**
I just added a Material Design timepicker to schedule alarm and wake me up everyday

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,15 +3,20 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.3.0'
 
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
```
Comment (naman14) [inline build.gradle]: Why add these 2 lines?

**Comment by naman14** [comment]
-You will need to rebase this on master, there have been a big code cleanup today
-There's already a PreferencesUtility class, keep all preferences related stuff there only. There's no need for another SharedPreferencesUtils class
- I don't think that this really needs to be added. Waiting for more opinions

**Comment by emersion** [comment]
Use your alarm clock (the _Clock_ app) to setup an alarm. Not a music player.

File: build.gradle (hunk 1 above)
Comment (TY-tianyu) [inline build.gradle]: these useless 2 lines has been removed before the last commit

**Comment by N07070** [comment]
I think it's not the role of the music player to wake me up. It's not in the respect of the UNIX philosophy code : Do one thing and to it well.

But even tho, I think this code is good work and I thank the author for it !

**Comment by naman14** [comment]
Thanks for the PR but as said, there really isn't any need for it. Feel free to send PR for other features

**Pull request closed without merging**